sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -221,7 +223,7 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       // file://path/to/data/files/employee.dat
       //
       // TODO: need a similar test for non-local mode.
-      if (local) {
+      if (local && !Utils.isWindows) {
```

Comment:
This is being skipped because `incorrectUri` below becomes `file://path/to/data/files/employee.dat` (or `file://C:/path/to/data/files/employee.dat` on Windows).
This seems checking if the file exists or not with `uri.getPath` assuming from [tables.scala#L223-L248](https://github.com/apache/spark/blob/5572ccf86b084eb5938fe62fd5d9973ec14d555d/sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala#L223-L248).

In Linux/Mac, it seems `uri.getPath` becomes `/to/data/files/employee.dat` where as `/path/to/data/files/employee.dat` on Windows.

The former path does not exists on Linux/Mac but the latter on Windows seems fine as the path seems correctly implicitly adding `C:`, meaning on Windows, these below seem fine:

```scala
new File("/C:/a/b/c").exists
new File("/a/b/c").exists
```

Therefore, the test below: 

```
intercept[AnalysisException] {
   sql(s"""LOAD DATA LOCAL INPATH "$incorrectUri" INTO TABLE non_part_table""")
}
```

dose not throw an exception on Windows because `incoorectURI` seems fine in this path.